content/docs/user-guide/experiment-management/comparing-experiments.md

```diff
@@ -0,0 +1,501 @@
+# Comparing Experiments
+
+You'll need a way to compare experiment results and track their corresponding
+parameters. DVC provides commands to list, tabulate and contrast them. Let's see
+how they can help you streamline the experimentation process.
+
+## List experiments in the workspace
+
+You can get a list of existing experiments in the repository with
+`dvc exp list`. Without any options, this command lists the experiments based on
+the latest commit of the current branch (Git `HEAD`).
+
+```dvc
+$ dvc exp list
+refs/tags/baseline-experiment:
+        cnn-64
+        cnn-128
+```
+
+If you want to list all the experiments in the repository regardless of their
+parent commit, use the `--all` flag.
+
+```dvc
+$ dvc exp list --all
+refs/tags/baseline-experiment:
+        cnn-64
+        cnn-128
+main:
+        exp-93150
+```
+
+## List experiments in another Git remote
+
+As we discuss in [Sharing Experiments], you can use `dvc exp push` to upload
+experiments to Git remotes. `dvc exp list` can be used to list the experiments
+in a Git remote.
+
+```dvc
+$ dvc exp list origin
+refs/tags/baseline-experiment:
+        cnn-32
+        cnn-64
+```
+
+This command lists the experiments originated from `HEAD`. You can add any other
```

Comment:
@iesahin another question here... do you know how `HEAD` is determined for a remote to list experiments from? Is it the local repo's HEAD? Is it the default branch HEAD always? We should probably clarify (let's add it to https://github.com/iterative/dvc.org/issues/2995).

Reply:
p.s. actually we could add it in https://github.com/iterative/dvc.org/pull/2908 as it's more about sharing than about comparing.

Reply:
Moved to https://github.com/iterative/dvc.org/pull/2908#pullrequestreview-798148999.